packages/core/admin/admin/src/hooks/useReleaseNotification/utils/api.js

```diff
@@ -1,13 +1,18 @@
-import axios from 'axios';
+import { getFetchClient } from '@strapi/helper-plugin';
 import packageJSON from '../../../../../package.json';
 
 const strapiVersion = packageJSON.version;
+const { get } = getFetchClient();
 
 const fetchStrapiLatestRelease = async () => {
   try {
     const {
       data: { tag_name },
-    } = await axios.get('https://api.github.com/repos/strapi/strapi/releases/latest');
+    } = await get('https://api.github.com/repos/strapi/strapi/releases/latest', {
```

Comment:
I'm wondering if we really should use the fetch client for external calls, because it is so specific to us and in the long-term I'd like to make it even more specific.

Why did you choose to replace it? Why not just use `fetch` here?

Reply:
it was just to have a common way to call APIs, I thought our aim was to use always useFetchClient and getFetchClient instead of axios. I also tried to understand if our hook was able to manage these kinds of calls and it is, but if we want just to handle internal calls is even better because I can avoid checking headers inside the hook and also check if the url is absolute

Reply:
refactored with fetch